packages/office-ui-fabric-react/src/components/Dialog/DialogContent.Props.ts

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { DialogContent } from './DialogContent';
+import { IButtonProps } from '../Button/Button.Props';
+
+export interface IDialogContent {
```

Comment:
yeah true...i guess that idea was to connect it to a componentRef, but never did.